doc/pages/administrator/030-installation_kubernetes.md

```diff
@@ -86,17 +86,67 @@ kubectl delete deployments.app -n kube-system astarte-operator
 **DO NOT** delete Astarte's CRDs! This will lead to the deletion of the entire Astarte deployment
 with a consequent data loss.
 
-To restore the Operator's functionalities with v1.0, all you have to do is following the install
-instructions, i.e.:
+Please, before proceeding to the next steps make sure the requirements listed in the
+[Prerequisites](./020-prerequisites.html#Prerequisites) section are satisfied (e.g. `cert-manager`
+installation).
+
+To restore the Operator's functionalities with v1.0, the first step is adding the Astarte Operator's
+Helm chart repository:
 ```bash
 helm repo add astarte https://helm.astarte-platform.org
 helm repo update
-helm install astarte-operator astarte/astarte-operator --version 1.0.0-alpha.1
 ```
 
-Please note that version `1.0.0-alpha1` is a safe landing version to perform the upgrade. After
-the successful installation, please upgrade to a more recent version following the instructions
-outlined in the [previous section](#upgrading-the-operator-v1-0).
+Then, let's set some environment variables that will come handy for the upcoming migration
+procedure:
++ `DIR` is the target directory in which the chart templates will be generated,
++ `RELEASE_NAME` is the name of the Astarte Operator deployment,
++ `RELEASE_NAMESPACE` is the namespace in which the Astarte Operator will reside.
+
+You can change the following values at your preference. However, the following values should be ok
+for the large majority of use cases.
+
+Note that you are responsible for checking that `RELEASE_NAMESPACE` exists within the cluster.
+```bash
+export DIR=/tmp
+export RELEASE_NAME=astarte-operator
+export RELEASE_NAMESPACE=kube-system
+```
+
+Generate the Helm templates with the following:
+```bash
+helm template $RELEASE_NAME astarte/astarte-operator --namespace $RELEASE_NAMESPACE \
+    --output-dir $DIR
+```
+The outcome of this instruction consists of a series of `yaml` files located in
+`$DIR/astarte-operator/templates`.
+
+In case of need, you can customize the generated templates according to the configurable options
+listed in the chart's values.
+
+Now it's time to prepare the cluster to allow the new operator installation relying on the templates
+generated in the previous step. This includes the installation/update of CRDs, the installation of
+RBACs, webhooks and the updating of labels and annotations. The process is automated and all you
+need to do is executing the following:
+```bash
+URL=https://raw.githubusercontent.com/matt-mazzucato/astarte-kubernetes-operator/add-hack-script/hack/upgrade-operator-011-10.sh
+curl -fsSL $URL > /tmp/upgrade-operator-011-10.sh
+bash /tmp/upgrade-operator-011-10.sh -d $DIR -n $RELEASE_NAME -N $RELEASE_NAMESPACE
+```
+
+Now it's time to install the Astarte Operator v1.0:
+```bash
+helm install $RELEASE_NAME astarte/astarte-operator -n $RELEASE_NAMESPACE --version 1.0.0 --skip-crds
+```
+As usual, you can customize the installation according to the configurable chart values.
```

Comment:
Again this should link somewhere where those values are mentioned or documented.